rewrite-core/src/test/java/org/openrewrite/remote/RemoteArchiveTest.java

```diff
@@ -58,10 +56,9 @@ void gradleWrapper(String version) throws Exception {
 
     @Test
     void gradleWrapperDownloadFails() throws Exception {
-        URL distributionUrl = requireNonNull(RemoteArchiveTest.class.getClassLoader().getResource("gradle-7.4.2-bin.zip"));
+        URL distributionUrl = new URL("http://example.com");
```

Comment:
That is a real website, however sparse, no need to have our tests ping someone's page.
```suggestion
        URL distributionUrl = new URL("http://example");
```